File: python/misc/date-and-time.py
from datetime import datetime
from datetime import timedelta

now = datetime.now()
print(now)
# 2018-06-15 18:23:51.500993

future = now + timedelta(12)
print(future)
# 2018-06-27 18:23:59.351647

print(now.year)
# 2018
print(now.month)
# 6
print(now.day)
# 15
print(now.hour)
# 18
print(now.minute)
# 23

# difference between dates
difference = future - now
print(difference.days)
# 12

File: _Scala/Primer/date-and-time.scala
import java.util.Calendar
import java.text.SimpleDateFormat

val now = Calendar.getInstance.getTime
// Sun Jun 17 19:18:49 CEST 2018

val year = new SimpleDateFormat("yyyy")
val month = new SimpleDateFormat("M")
val day = new SimpleDateFormat("d")
val hour = new SimpleDateFormat("H")
val minute = new SimpleDateFormat("m")

year.format(now)                            // 2018
month.format(now)                           // 6
day.format(now)                             // 17
hour.format(now)                            // 19
minute.format(now)                          // 18